<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>743</width>
    <height>557</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout" rowstretch="0,1">
   <item row="1" column="0" colspan="5">
    <widget class="QGroupBox" name="plot_groupBox">
     <property name="title">
      <string>Plot</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2"/>
    </widget>
   </item>
   <item row="0" column="0">
    <widget class="QGroupBox" name="groupBox_scancontrol">
     <property name="title">
      <string>Andor CCD</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_4">
      <item row="1" column="4">
       <widget class="QLabel" name="andor_ccd_status_label">
        <property name="text">
         <string>STATUS</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="QLabel" name="label_22">
        <property name="text">
         <string>Temperature</string>
        </property>
       </widget>
      </item>
      <item row="4" column="1">
       <widget class="QCheckBox" name="save_h5_checkBox">
        <property name="text">
         <string>Save H5</string>
        </property>
       </widget>
      </item>
      <item row="4" column="2">
       <widget class="QCheckBox" name="andor_ccd_continuous_checkBox">
        <property name="text">
         <string>continuous</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QLabel" name="label_21">
        <property name="text">
         <string>EM Gain</string>
        </property>
       </widget>
      </item>
      <item row="1" column="2">
       <widget class="QDoubleSpinBox" name="andor_ccd_int_time_doubleSpinBox">
        <property name="keyboardTracking">
         <bool>false</bool>
        </property>
        <property name="decimals">
         <number>3</number>
        </property>
        <property name="minimum">
         <double>0.001000000000000</double>
        </property>
        <property name="maximum">
         <double>999.990000000000009</double>
        </property>
        <property name="singleStep">
         <double>0.100000000000000</double>
        </property>
        <property name="value">
         <double>1.000000000000000</double>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QLabel" name="label">
        <property name="text">
         <string>Exposure Time (s)</string>
        </property>
       </widget>
      </item>
      <item row="3" column="3">
       <widget class="QDoubleSpinBox" name="andor_ccd_temp_doubleSpinBox">
        <property name="focusPolicy">
         <enum>Qt::NoFocus</enum>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
        <property name="readOnly">
         <bool>true</bool>
        </property>
        <property name="buttonSymbols">
         <enum>QAbstractSpinBox::NoButtons</enum>
        </property>
        <property name="suffix">
         <string> C</string>
        </property>
        <property name="decimals">
         <number>0</number>
        </property>
        <property name="minimum">
         <double>-300.000000000000000</double>
        </property>
        <property name="maximum">
         <double>999.000000000000000</double>
        </property>
       </widget>
      </item>
      <item row="3" column="2">
       <widget class="QDoubleSpinBox" name="temp_setpoint_doubleSpinBox"/>
      </item>
      <item row="0" column="1" colspan="2">
       <widget class="QCheckBox" name="hw_connect_checkBox">
        <property name="text">
         <string>Hardware Connect</string>
        </property>
       </widget>
      </item>
      <item row="3" column="4">
       <widget class="QLabel" name="temp_status_label">
        <property name="text">
         <string>STATUS</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item row="2" column="2">
       <widget class="QDoubleSpinBox" name="andor_ccd_emgain_doubleSpinBox">
        <property name="keyboardTracking">
         <bool>false</bool>
        </property>
        <property name="decimals">
         <number>0</number>
        </property>
        <property name="maximum">
         <double>4096.000000000000000</double>
        </property>
        <property name="singleStep">
         <double>1.000000000000000</double>
        </property>
        <property name="value">
         <double>100.000000000000000</double>
        </property>
       </widget>
      </item>
      <item row="4" column="3">
       <widget class="QPushButton" name="andor_ccd_start_pushButton">
        <property name="text">
         <string>Start</string>
        </property>
       </widget>
      </item>
      <item row="4" column="4">
       <widget class="QPushButton" name="andor_ccd_interrupt_pushButton">
        <property name="text">
         <string>Interrupt</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>Background</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="3" column="0">
       <widget class="QPushButton" name="andor_ccd_abort_bg_pushButton">
        <property name="text">
         <string>Abort BG</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QPushButton" name="andor_ccd_acq_bg_pushButton">
        <property name="text">
         <string>Acq Background</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QCheckBox" name="andor_ccd_shutter_open_checkBox">
        <property name="text">
         <string>Shutter Open</string>
        </property>
        <property name="tristate">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item row="4" column="0">
       <widget class="QCheckBox" name="andor_ccd_bgsub_checkBox">
        <property name="text">
         <string>BG Substract</string>
        </property>
       </widget>
      </item>
      <item row="5" column="0">
       <spacer name="verticalSpacer_2">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </item>
   <item row="0" column="4">
    <spacer name="horizontalSpacer">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>40</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="0" column="2">
    <widget class="QGroupBox" name="groupBox_2">
     <property name="title">
      <string>Display</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_3">
      <item row="4" column="0">
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>Horizontal Axis</string>
        </property>
       </widget>
      </item>
      <item row="5" column="0">
       <widget class="QComboBox" name="wl_calib_comboBox"/>
      </item>
      <item row="3" column="0">
       <spacer name="verticalSpacer">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
